packages/emotion-theming/types/tests.tsx

```diff
@@ -11,12 +13,11 @@ declare const theme: Theme
 
 interface Props {
   prop: boolean
-  theme: Theme
 }
-declare const CompSFC: React.SFC<Props>
-declare class CompC extends React.Component<Props> {}
+declare const CompSFC: React.FC<Props>
```

Comment:
the declared identifier could be renamed as well

Reply:
Done, and renamed a few other instances